android/app/src/main/java/com/microsoft/codepush/common/core/CodePushState.java

```diff
@@ -0,0 +1,36 @@
+package com.microsoft.codepush.common.core;
+
+import com.microsoft.codepush.common.enums.CodePushInstallMode;
+
+public class CodePushState {
```

Comment:
Javadoc missing.

Reply:
@annakocheshkova thanks, I will fix that.